app/config/services/membership.xml

```diff
@@ -45,5 +45,15 @@
             <tag name="form.type" />
         </service>
 
+        <service id="app.membership.nearby_committee_provider" class="AppBundle\Membership\NearbyCommitteeProvider" public="false">
```

Comment:
I asked myself the same question... I'm shared between these options. What do you prefer?